src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/Controllers/AuthController.cs

```diff
@@ -83,11 +88,49 @@ public async Task SendEmailConfirmLink(SendEmailConfirmLinkRequestDto sendConfir
         var result = await _fluentEmail
             .To(user.Email, user.DisplayName)
             .Subject(EmailStrings.ConfirmationLinkEmailSubject)
-            .UsingTemplateFromEmbedded("TodoTemplate.Api.Resources.EmailConfirmation.cshtml",
-                                    new EmailConfirmationModel
+            .UsingTemplateFromEmbedded($"TodoTemplate.Api.Resources.EmailConfirmation.cshtml",
+                                    new EmailLinksModel
                                     {
                                         DisplayName = user.DisplayName,
-                                        ConfirmationLink = confirmationLink
+                                        Link = confirmationLink
+                                    },
+                                    assembly)
+            .SendAsync(cancellationToken);
+
+        if (!result.Successful)
+            throw new ResourceValidationException(result.ErrorMessages.ToArray());
+    }
+
+    [HttpPost("[action]")]
+    public async Task SendEmailForgotPasswordLink(SendEmailForgotPasswordLinkRequestDto sendConfirmLinkEmailRequest, CancellationToken cancellationToken)
+    {
+        var user = await _userManager.FindByEmailAsync(sendConfirmLinkEmailRequest.Email);
+
+        if (user is null)
+            throw new BadRequestException(nameof(ErrorStrings.UserNameNotFound));
+
+        var token = await _userManager.GeneratePasswordResetTokenAsync(user);
+
+        var tokenBase64 = Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(token));
+
+        var resetPasswordLink = $"reset-password?email={user.Email}&token={tokenBase64}";
+
+#if BlazorServer
+        resetPasswordLink = $"{_appSettings.WebServerAddress}{resetPasswordLink}";
+#else
```

Comment:
The relative url would be enough, no need to #else

Reply:
I'm not sure about that, the generated link has been sent to the user by email just 2 lines after. So, we need an absolute url. If there is another issue please explain more. @ysmoradi